tests/testthat/test-metadata_to_coldata.R

```diff
@@ -0,0 +1,160 @@
+# establish single test sce
+sce <- sim_sce(n_genes = 20, n_cells = 100, n_empty = 0)
+metadata(sce)$sample_id <- "sample_id"
+metadata(sce)$library_id <- "library_id"
+
+# create sample data frame
+sample_metadata_df <- data.frame(
+  sample_id = "sample_id",
+  library_id = "library_id",
+  diagnosis = "diagnosis"
```

Comment:
Just because I started to get confused here, can we change these to different values from the column name? (Note that tests will need to be changed too)
```suggestion
  sample_id = "sample1",
  library_id = "library1",
  diagnosis = "diagnosis1"
```